<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>attendanceDialog</class>
 <widget class="QDialog" name="attendanceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>580</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>580</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>580</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>attendanceDialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">#widget{
border-image: url(:/masterDesktop/Resources/complaints_bg.png);
}
#btn_close{
border-image: url(:/masterDesktop/Resources/close.png);
}
#btn_close:pressed{
margin:1px;
}
#wgt_title{
background:#002E29;
border:1px solid #00947E;
}
#lbl_title{
font-family:&quot;微软雅黑&quot;;
font-size:18px;
color:#FFFFFF;
}</string>
     </property>
     <widget class="QPushButton" name="btn_close">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>28</y>
        <width>37</width>
        <height>37</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>37</width>
        <height>37</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>37</width>
        <height>37</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>414</width>
        <height>434</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QWidget" name="wgt_title" native="true">
         <property name="minimumSize">
          <size>
           <width>412</width>
           <height>42</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>412</width>
           <height>42</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="lbl_title">
            <property name="text">
             <string>本月签到</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <property name="minimumSize">
          <size>
           <width>412</width>
           <height>380</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>412</width>
           <height>380</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">#tableWidget{
background:transparent;
border:none;
font-family:&quot;微软雅黑&quot;;
font-size:16px;
}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>9</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>6</number>
          </property>
          <item>
           <widget class="QTableWidget" name="tableWidget">
            <property name="styleSheet">
             <string notr="true">/* QTableView {
      selection-background-color: transparent;
  }*/

QTableView::item:hover{
  color:#FFFFFF;
}
 QTableView::item:selected {
      border: 1px solid #6a6ea9;
color:#207C78;
  }</string>
            </property>
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="showGrid">
             <bool>false</bool>
            </property>
            <property name="rowCount">
             <number>7</number>
            </property>
            <property name="columnCount">
             <number>7</number>
            </property>
            <attribute name="horizontalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>58</number>
            </attribute>
            <attribute name="horizontalHeaderMinimumSectionSize">
             <number>25</number>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderDefaultSectionSize">
             <number>60</number>
            </attribute>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <column/>
            <column/>
            <column/>
            <column/>
            <column/>
            <column/>
            <column/>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>11</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="minimumSize">
               <size>
                <width>78</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>78</width>
                <height>30</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
font-family:&quot;微软雅黑&quot;;
font-size:16px;
color:#151515;
background:#2EB4AF;
border:0px;
border-radius:2px;
}
QPushButton:pressed{
margin:1px;
}</string>
              </property>
              <property name="text">
               <string>我知道了</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true">font-family:&quot;微软雅黑&quot;;
font-size:12px;
color:#ACACAC;</string>
            </property>
            <property name="text">
             <string>每天连续上网半小时后会自动签到</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="masterDesktop.qrc"/>
 </resources>
 <connections/>
</ui>
